suitcase/mongo_embedded/__init__.py

```diff
@@ -186,7 +209,7 @@ def _event_worker(self):
             do_push = False
             try:
                 if event is None:
-                    event = self._event_queue.get(timeout=0.5)
+                    event = self._event_queue.get(timeout=0.2)
```

Comment:
It's best to declare magic numbers like this as constants at the top of the module so that it's easy to quickly review the tunable parameters. See example: https://github.com/NSLS-II/caproto/blob/60327cda7d06203e1bf8db69d6e5014ae1276c2b/caproto/threading/client.py#L152-L160